pdal/util/Utils.cpp

```diff
@@ -81,25 +81,6 @@ double Utils::random(double minimum, double maximum)
 }
 
 
-double Utils::uniform(const double& minimum, const double& maximum,
```

Comment:
Do any filters still use this? I suppose this is pre-c++11 stuff.

 It seems out of scope to remove these capabilities in this PR, or the description should be updated to reflect we're removing them.

Reply:
They were only used by filters.faux and they were broken, because you had to set a seed for each call, which was dumb.  I probably wrote them.